### PR TITLE
Sprite Lab event tests

### PR DESCRIPTION
**Pull request by ajpal**
Test coverage for click, key, and collision events in Sprite Lab

Follow up- will add more test coverage for costume group collisions as part of https://codedotorg.atlassian.net/browse/STAR-537

### CODE REVIEW

File: apps/test/unit/gamelab/spritelab/spriteUtilsTest.js
Comment (jmkulwik) [inline apps/test/unit/gamelab/spritelab/spriteUtilsTest.js]: Nit: remove newline

File: apps/test/unit/gamelab/spritelab/spriteUtilsTest.js
Comment (jmkulwik) [inline apps/test/unit/gamelab/spritelab/spriteUtilsTest.js:249]: NIT: 'work with individual sprites'

Comment (jmkulwik) [reply apps/test/unit/gamelab/spritelab/spriteUtilsTest.js:249]: Same with others below

Comment (ajpal) [reply apps/test/unit/gamelab/spritelab/spriteUtilsTest.js:249]: Why "work"? I was trying to make it read like "it works with..."

**Review by jmkulwik — APPROVED** [review]
LGTM! Had a couple nits in there. No need for a second review if you do/don't change them.

**Review by joshlory — APPROVED** [review]
This is great!

**Pull request merged**
